AzureData/Source/AzureData.swift

```diff
@@ -415,6 +415,10 @@ public func get (userWithId userId: String, inDatabase database: Database, callb
 }
 
 // delete
+public func delete (_ user: User, fromDatabase database: Database, callback: @escaping (DataResponse) -> ()) {
```

Comment:
this override isn't necessary as we have the the generic `DocumentClient.default.delete(Resource)`